pdctl/command/global.go

```diff
@@ -122,6 +121,8 @@ func printResponseError(r *http.Response) {
 	io.Copy(os.Stdout, r.Body)
 }
 
+/* unused
```

Comment:
please clean the unused code.

Reply:
I am concerned that `testCheckStores` is not being run by accident. I would like for @disksing to confirm that code should be removed (in this PR or send a new PR later).

Reply:
Okay, the code is removed now.